doobie/src/main/scala/za/co/absa/db/fadb/doobie/postgres/circe/implicits/package.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.db.fadb.doobie.postgres.circe
+
+import cats.Show
+import cats.data.NonEmptyList
+import doobie.{Get, Put}
+import io.circe.Json
+import org.postgresql.jdbc.PgArray
+import org.postgresql.util.PGobject
+import io.circe.parser._
+
+import scala.util.{Failure, Success, Try}
+
+package object implicits {
```

Comment:
also, what about JSON/JSONB put and get? (these here are List of JSON/JSONB, so I'm talking about the non-array variant)

Reply:
the non-array variant is already available, see the docs https://github.com/tpolecat/doobie/blob/v1.0.0-RC2/modules/docs/src/main/mdoc/docs/15-Extensions-PostgreSQL.md

Reply:
hmm okay! Thanks

Reply:
I've made it available also in the implicits object so we can import everything from the same place.